### ISSUE TITLE
Integrate API key signup on the site instead of linking off

### ISSUE THREAD

**Issue by gbinal** [18F/openFEC#2231]
Howdy folks.  

**TL;DR** - we'd like to make a small change so that the api key registration form is integrated into the site instead of being a link off to the api.data.gov page.  This will help third party developers connect with you more directly when they have questions or need to get in touch.  

#### Background

Right now, developers [are referred](https://api.open.fec.gov/developers/) to [this page](https://api.data.gov/signup/) to get an API key.  This works but has several downsides.   

#### Why?  

A few reasons: 
* This takes the developer away from the FEC site and out of the developer hub experience they were in.  
* On the api.data.gov page, there's a whole other set of links and information that isn't relevant for the FEC developer and may confuse them.  
* Also, the contact us links there will take them to the api.data.gov team, not the FEC team.    

#### How to resolve

Good news!  This is easy to address.  😄    There are [instructions for embedding the signup form](https://github.com/18F/api.data.gov/wiki/User-Manual:-Agencies#embedding-the-api-key-signup-form-on-your-own-documentation-site) and we're happy to help if you have any problems.  

A few thoughts: 
* It looks like the API documentation is all on one page.  One option is to create another page like you see [here](https://gsa.github.io/sam_api/sam/key), [here](https://developer.nrel.gov/signup/), [here](https://regulationsgov.github.io/developers/key/), and [here](https://usedgov.github.io/key/).  But if you don't want to do that, it could go at the bottom of the page (like [this](https://api.nasa.gov/index.html#apply-for-an-api-key)), with a the link just going down to that section or it could be made a popup, like [this](https://ndb.nal.usda.gov/ndb/doc).  
* Wherever you put it, it's particularly important to modify the code to customize the `registration source`, `exampleApiUrl`, `contactUrl`, and `emailFromName` (example [here](https://github.com/GSA/sam_api/pull/108/files).  This will ensure that developers that want to get in touch reach you instead of the api.data.gov team.  

Thanks for your help with this.  It's a small thing but best for all to go ahead and address.  Holler if you need any help or have any questions.

**Comment by LindsayYoung** [comment]
Would April or May be a reasonable time to get to this? This month, we are migrating to gov cloud, getting ATOed and making the beta site the main site. 

This is part of the API documentation improvements we have in icebox for now: https://github.com/18F/openFEC/issues/1725

**Comment by gbinal** [comment]
Totally reasonable.  It's just definitely worth getting to at some point b/c we'll be handing along developer queries with a regret delay in the meantime.
